pyre/bench/synth/wrapper_function_invalidation.py

```
@@ -0,0 +1,104 @@
+# type's version tag and leaves the descriptor at the same address, so a fold
+# that pinned the DESCRIPTOR still holds while the callable inside it has
+# already been replaced.
+#
+# Each rebind happens INSIDE its loop: a read after the loop is interpreted and
+# would not consult what the trace baked.  The wrapped bodies are residual-free
+# so the folds stand rather than aborting.
+N = 400000
+SWITCH = N // 2
+
+
+def first_hook(name):
```

Ruff (0.16.1)

[warning] Unused function argument: `name`

(ARG001)